rosbag2_cpp/src/rosbag2_cpp/reader.cpp

```diff
@@ -40,8 +40,6 @@ void Reader::open(const std::string & uri)
   rosbag2_cpp::StorageOptions storage_options;
   storage_options.uri = uri;
   storage_options.storage_id = "sqlite3";
-  storage_options.max_bagfile_size = 0;  // default
```

Comment:
I believe this assignment in unnecessary and keeps the code clean and consistent 

[Note: if we want to keep this then storage_options.max_bagfile_duration = 0, is something we should add to this.]

Reply:
Yes - I think you are right. It might also make sense to move the "sqlite3" default value to the struct definition as well. No need to do that right now though